v2/charts/azure-service-operator/values.yaml

```diff
@@ -0,0 +1,50 @@
+# Create secret or use an existing secret
+createAzureOperatorSecret: true
+
+# azureTenantID is the AAD tenant that the subscription is in
+azureTenantID: ""
+# azureSubscriptionID is the Azure Subscription the operator will act against.
+azureSubscriptionID: ""
+
+# cloudEnvironment is the cloud environment. Possible values include: AzurePublicCloud, AzureUSGovernmentCloud, AzureChinaCloud, AzureGermanCloud
+cloudEnvironment: AzurePublicCloud
+
+# azureClientID is the client ID of the Azure Service Principal or Managed Identity to use to authenticate with Azure.
+azureClientID: ""
+
+# azureClientSecret is the client secret of the Azure Service Principal used to authenticate with Azure.
+# This is required when using Service Principal authentication.
+azureClientSecret: ""
+
+# SyncPeriod is the frequency at which resources are re-reconciled with Azure
+# when there have been no triggering changes in the Kubernetes resources. This sync
+# exists to detect and correct changes that happened in Azure that Kubernetes is not
+# aware about. BE VERY CAREFUL setting this value low - even a modest number of resources
+# can cause subscription level throttling if they are re-synced frequently.
+# If nil, no sync is performed. Durations are specified as "1h", "15m", or "60s". See
+# https://pkg.go.dev/time#ParseDuration for more details.
+azureSyncPeriod: ""
+
+# TargetNamespaces lists the namespaces the operator will watch
+# for Azure resources (if the mode includes running watchers). If
+# it's empty the operator will watch all namespaces.
+azureTargetNamespaces: []
+
+# OperatorMode determines whether the operator should run
+# watchers, webhooks or both.
+azureOperatorMode: ""
+
+# api version to use for cert-manager custom resources. Depends on the version of cert-manager which is deployed externally to this chart
+certManagerResourcesAPIVersion: cert-manager.io/v1
```

Comment:
I actually think you probably don't need this. Now that cert-manager has `v1` I don't think we want people going back to `v1beta` or `v1alpha`. In ASOv1 we do for legacy reasons, but since ASOv2 is more greenfield I think we shouldn't bother.

Reply:
Sweet! I'll remove it if there's no use of it.

Reply:
Sweet! I'll remove it if there's no use of it.